jwst/rscd/rscd_sub.py

```diff
@@ -185,12 +291,28 @@ def do_correction(input_model, rscd_model):
 
 def get_rscd_parameters(input_model, rscd_model):
 
+
     """
     Read in the parameters from the reference file
-    Seperate these parameters based on even and odd rows
-
     Store the parameters in a param dictionary
+
+    Parameters
+    ----------
+    input_model: data model object
+        science data to be corrected
+
+    rscd_model: rscd model object
```

Comment:
Same here for using "~jwst.datamodels.RSCDModel" instead of "rscd model object"

Reply:
Done.